api.go

```diff
@@ -102,6 +103,40 @@ func (a *API) NumSlots() (int, error) {
 	return n, unmarshalPyON(s, &n)
 }
 
+// Uptime returns FAH uptime
+func (a *API) Uptime() (time.Duration, error) {
+	// work around for response to uptime command not including a trailing newline
+	s, err := a.Exec("eval \"$(uptime)\\n\"")
+	if err != nil {
+		return time.Duration(0), err
+	}
+
+	// when formatting with a newline, the response contains an extra trailing backslash
+	u, err := parseFAHDuration(strings.TrimSuffix(s, "\\"))
+	if err != nil {
+		return time.Duration(0), err
+	}
+
+	return u, nil
+}
+
+// Date returns the current date and time
+func (a *API) Date() (time.Time, error) {
```

Comment:
Although the wrapper should cover as much of the underlying API as possible, I hardly think anyone will need this. If they do, they should just use `Exec()`. I don't think any of the "Standard Commands" should be added. Please remove `Date()`.

Reply:
For context, the reason I'm interested in this library is to build a F@H prometheus exporter. Being new to Go, I based it on the memcached one, which exported a metric for the current server time. But I agree, I won't be using that metric in my dashboards, and it makes sense not to wrap the "Standard Commands". Will remove.

Reply:
I admit I have the "you don't need it" mindset, but now that you mentioned the use case I see a good reason to keep it. Sorry.

Then comes the question of "should we implement `add <number> <number>`"? My answer would probably be: send a PR.

Reply:
I'm also pretty minimalistic, definitely no need to apologize. I think the stance of not wrapping the "Standard Commands" makes sense, and I don't actually understand why the telnet API exposes a scripting language.

For now, why don't we not wrap `date`, but export the helper for the `exec` hack? That way I can easily implement support for it in the application code -- it's just one `time.Parse` call. If in the future, there is a good reason to support the rest of the Standard Commands in this library, I can always switch to the library version.

(As a side note, I do need the library to support the `uptime` command, because I need the special FAH duration parser which is not exported.)

Reply:
Sounds good!